integration/nwo/fabric/topology/core_template.go

```diff
@@ -310,6 +310,10 @@ fabric:
         type: file
         opts:
           path: {{ FSCNodeVaultPath }}
+      txidstore:
```

Comment:
do we need this also for orion?

Reply:
Orion does not use a txidstore cache at all! So if we want it let's create an issue for that

Reply:
I see, that's true. All right. Let's address it elsewhere.